README.md

```diff
@@ -14,14 +14,16 @@ Summary: Massively spiffy yet delicately unobtrusive compression library
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/zlib-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/zlib-feedstock)
-OSX: [![TravisCI](https://travis-ci.org/conda-forge/zlib-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/zlib-feedstock)
-Windows: [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/conda-forge/zlib-feedstock?svg=True)](https://ci.appveyor.com/project/conda-forge/zlib-feedstock/branch/master)
+[![Linux](https://img.shields.io/circleci/project/github/conda-forge/zlib-feedstock.svg?label=linux)](https://circleci.com/gh/conda-forge/zlib-feedstock)
+[![OSX](https://img.shields.io/travis/conda-forge/zlib-feedstock.svg?label=OSX)](https://travis-ci.org/conda-forge/zlib-feedstock)
+[![Windows](https://img.shields.io/appveyor/ci/conda-forge/zlib-feedstock.svg?label=Windows)](https://ci.appveyor.com/project/conda-forge/zlib-feedstock/branch/master)
```

Comment:
These new badges seem to have different statuses than the previous ones.

Reply:
Should we just revert this part? We can always add it back once someone has time to fix them.